docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,552 @@
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "activations = project.list_alert_activations(\n",
+    "    entity=\"test-func-handler.db80cba0c4be4ee9b86a09cf12a89991\"\n",
+    ")\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15c4642b-2fac-4bbf-b69d-7769e087240b",
+   "metadata": {},
+   "source": [
+    "## Filter by since/until\n",
+    "\n",
+    "`since` and `until` should be passed as `datatime.datatime` objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "e2a5713e-67a5-4ea9-a113-fd9ec699a7ae",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "activations = project.list_alert_activations(since=datetime.datetime.now())\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "f1cf11f6-6521-456c-b38a-0eb44b00d8c7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "activations = project.list_alert_activations(\n",
+    "    since=datetime.datetime(2024, 12, 11, 10, 5, 17, 674000),\n",
+    "    until=datetime.datetime.now(),\n",
+    ")\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "be0c33ac-a2a8-40fc-bebe-e36818fc9df7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "alert_config = run_db.list_alerts_configs()[0]\n",
+    "activations = alert_config.list_activations()\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "827f02f7-e60a-4a29-82d6-c818dae1a3b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# update alert config and get activations since last update\n",
+    "alert_config = project.store_alert_config(alert_config)\n",
+    "activations = alert_config.list_activations(from_last_update=True)\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32992696-e584-4b34-bcd7-320c613eebad",
+   "metadata": {},
+   "source": [
+    "## Group by attributes\n",
+    "The `group_by` method organizes alert activations into a dictionary based on specified attributes, making it easier to analyze or filter data.\n",
```

Comment:
"The `group_by` method organizes alert activations into a dictionary based on specified attributes, making it easier to analyze or filter data.\n",
    "It groups activations by one or more attributes, such as project or severity, etc., with dictionary keys as tuples of the attribute values and values as lists of activations.\n",
    "This is especially useful for processing activations by categories, such as identifying alerts by their severity level or grouping them by projects, which can result in detailed insights."